ui/src/components/ValidatorDetails/Details.tsx

```diff
@@ -245,6 +245,22 @@ export function Details({ validator }: DetailsProps) {
                   />
                 </dd>
               </div>
+
+              {validator.config.maxAlgoPerPool > 0n && (
+                <div className="py-4 grid grid-cols-[2fr_3fr] gap-4 xl:grid-cols-2">
+                  <dt className="text-sm font-medium leading-normal text-muted-foreground">
+                    Max Stake Per Pool
+                  </dt>
+                  <dd className="flex items-center justify-between gap-x-2 text-sm leading-normal">
+                    <AlgoDisplayAmount
+                      amount={validator.config.maxAlgoPerPool}
```

Comment:
we'll want to have it show the lesser of maxAlgoPerPool or 15% of online stake / num of pools

Reply:
Updated the implementation to show the effective max, not just the configured max (if set): [`9502a49` (#370)](https://github.com/algorandfoundation/reti/pull/370/commits/9502a497c1107cbdb270dcd59c7a8de9769c08cb)